packages/patternfly-3/patternfly-react-extensions/src/components/CatalogTile/CatalogTile.stories.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info/dist/index';
+import { defaultTemplate } from 'storybook/decorators/storyTemplates';
+import { storybookPackageName } from 'storybook/constants/siteConstants';
+import pfBrand from './helpers/patternfly.svg';
```

Comment:
so similar to `'patternfly/dist/img/logo.svg' patternfly svg, but a tad darker... i guess this is more appealing on white backgrounds. Maybe we should export this svg from pf react for now? (totally optional though)

Reply:
I'll swap to using the patternfly/dist image. Thanks.